src/main/java/org/jitsi/protocol/xmpp/AbstractOperationSetJingle.java

```diff
@@ -134,8 +163,24 @@ public boolean initiateSession(boolean useBundle,
             inviteIQ.addExtension(startMutedExt);
         }
 
-        IQ reply = (IQ) getConnection().sendPacketAndGetReply(inviteIQ);
+        return inviteIQ;
+    }
 
+    /**
+     * Checks based on supplied <tt>reply</tt> IQ value whether or not given
+     * <tt>JingleSession</tt> has been accepted by the client.
+     *
+     * @param session <tt>JingleSession</tt> instance for which we're evaluating
+     * the response value.
+     * @param reply <tt>IQ</tt> response to Jingle invite IQ or <tt>null</tt> in
+     * case of timeout.
+     *
+     * @return <tt>true</tt> if invite IQ for which we're evaluating the value
+     * of response packet is considered accepted or <tt>false</tt> otherwise.
```

Comment:
yes, response packet which is type of IQ(as in smack types), by <tt>IQ</tt> I meant the type of argument, does it sense ?